server/config/cloudinaryconfig.js

```diff
@@ -0,0 +1,49 @@
+import multer from 'multer';
+import cloudinary from 'cloudinary';
+import cloudinaryStorage from 'multer-storage-cloudinary';
+import dotenv from 'dotenv';
+import Response from '../utils/response.util';
+
+
+dotenv.config();
+
+cloudinary.config({
+  cloudinary_name: process.env.CLOUDINARY_URL,
+});
+
+const storage = cloudinaryStorage({
+  cloudinary,
+  folder: 'forsetti',
```

Comment:
can you make the folder action specific? i.e there should be a folder specifically for article images, so there can also be another for user images

Reply:
> can you make the folder action specific? i.e there should be a folder specifically for article images, so there can also be another for user images

No problem.